I get the following error:

```
Changing application language to en-us
Fetching translation json for locale: en-us
Could not fetch translation json for locale: 'en-us' Error: Cannot find module './i18n/strings/en-us.json'
Require stack:
- /nix/store/rfsgqqv6hxrkq7k9zqxfg2v5gnwmi4ir-element-desktop-1.7.33/share/element/electron/lib/language-helper.js
- /nix/store/rfsgqqv6hxrkq7k9zqxfg2v5gnwmi4ir-element-desktop-1.7.33/share/element/electron/lib/tray.js
- /nix/store/rfsgqqv6hxrkq7k9zqxfg2v5gnwmi4ir-element-desktop-1.7.33/share/element/electron/lib/electron-main.js
- /nix/store/kjv95jyji0a6587l7ljnnpb56vlb2iha-electron-13.1.6/lib/electron/resources/default_app.asar/main.js
-
    at Module._resolveFilename (internal/modules/cjs/loader.js:887:15)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:257:1128)
    at Module._load (internal/modules/cjs/loader.js:732:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at AppLocalization.fetchTranslationJson (/nix/store/rfsgqqv6hxrkq7k9zqxfg2v5gnwmi4ir-element-desktop-1.7.33/share/element/electron/lib/language-helper.js:76:20)
    at /nix/store/rfsgqqv6hxrkq7k9zqxfg2v5gnwmi4ir-element-desktop-1.7.33/share/element/electron/lib/language-helper.js:89:39
    at Array.forEach (<anonymous>)
    at AppLocalization.setAppLocale (/nix/store/rfsgqqv6hxrkq7k9zqxfg2v5gnwmi4ir-element-desktop-1.7.33/share/element/electron/lib/language-helper.js:88:17) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/nix/store/rfsgqqv6hxrkq7k9zqxfg2v5gnwmi4ir-element-desktop-1.7.33/share/element/electron/lib/language-helper.js',
    '/nix/store/rfsgqqv6hxrkq7k9zqxfg2v5gnwmi4ir-element-desktop-1.7.33/share/element/electron/lib/tray.js',
    '/nix/store/rfsgqqv6hxrkq7k9zqxfg2v5gnwmi4ir-element-desktop-1.7.33/share/element/electron/lib/electron-main.js',
    '/nix/store/kjv95jyji0a6587l7ljnnpb56vlb2iha-electron-13.1.6/lib/electron/resources/default_app.asar/main.js',
    undefined
  ]
}
```
which causes
![image](https://user-images.githubusercontent.com/16734772/126814103-3fb1971f-c1d9-4b8d-9caa-7a9d6fb9c435.png)

--- a/pkgs/applications/networking/instant-messengers/element/element-desktop.nix
+++ b/pkgs/applications/networking/instant-messengers/element/element-desktop.nix
@@ -29,6 +29,8 @@ in mkYarnPackage rec {
     export HOME=$(mktemp -d)
     pushd deps/element-desktop/
     npx tsc
+    yarn run i18n
+    node ./scripts/copy-res.js
     popd
     runHook postBuild
   '';
@@ -41,6 +43,7 @@ in mkYarnPackage rec {
     cp -r './deps/element-desktop/res/img' "$out/share/element"
     rm "$out/share/element/electron/node_modules"
     cp -r './node_modules' "$out/share/element/electron"
+    cp $out/share/element/electron/lib/i18n/strings/en_EN.json $out/share/element/electron/lib/i18n/strings/en-us.json
 
     # icons
     for icon in $out/share/element/electron/build/icons/*.png; do

